angularjs-1.5-basics/gulp-tasks/build.js

```diff
@@ -0,0 +1,247 @@
+'use strict';
+
+var gulp = require('gulp');
+var $ = require('gulp-load-plugins')();
```

Comment:
__[jshint]__
* Redefinition of '$'.
* 'require' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->